x/group/client/cli/prompt.go

```diff
@@ -31,15 +31,18 @@ type proposalType struct {
 
 // Prompt the proposal type values and return the proposal and its metadata.
 func (p *proposalType) Prompt(cdc codec.Codec) (*Proposal, govtypes.ProposalMetadata, error) {
-	proposal := &Proposal{}
 
 	// set metadata
 	metadata, err := govcli.Prompt(govtypes.ProposalMetadata{}, "proposal")
 	if err != nil {
 		return nil, metadata, fmt.Errorf("failed to set proposal metadata: %w", err)
 	}
 	// the metadata must be saved on IPFS, set placeholder
```

Comment:
```suggestion

```

Reply:
Will do in as follow-up as I need to touch this file for supporting slice in prompts